docs/QuickStart.md

```diff
@@ -0,0 +1,159 @@
+# QuickStart
+
+## Installation
+
+Two ways for installation: you can get NNI from pypi or clone the repo and install it from there.
+
+**Install through pip**
+
+* We support Linux and MacOS in current stage, Ubuntu 16.04 or higher, along with MacOS 10.14.1 are tested and supported. Simply run the following `pip install` in an environment that has `python >= 3.5`.
+
+```bash
+    python3 -m pip install --user --upgrade nni
+```
+
+* Note:
+  * If you are in docker container (as root), please remove `--user` from the installation command.
+  * If there is any error like `Segmentation fault`, please refer to [FAQ][1]
+
+**Install through source code**
```

Comment:
shall we remove install from source code in quick start??

Reply:
Good suggestion, I also thought about adding this is too complicated.
My consideration is that in the outline of our documentation, this is the only place to mention installation, so I list all the available download methods here.